2023-09-04.mdx

```diff
@@ -0,0 +1,39 @@
+- **File Attachments are now available** - We have embedded [file attachments](https://xata.io/file-attachments) directly into our serverless database and improved the experience for building data apps that require binary storage.
+- **New UI experience for CSV import and export** - Introducing an entirely new [CSV import and export](https://xata.io/docs/csv-data/import-data) interface that simplifies your Xata.io experience, enabling you to automatically create new tables by directly importing your data into Xata's database. The export feature allows you to download your data as CSV files.
+- **Support for JSON ingestion** - Basic support for [JSON column](https://xata.io/blog/json-column-type) types is now available, expanding the range of data formats that Xata.io can handle. 
+- **The Xata Python SDK is in GA** - Our [Python SDK](https://xata.io/docs/sdk/python/overview) is now in general availability. We're excited to release version 1.0.0 of our Python SDK, offering a more stable and feature-rich experience for developers.
+- **Direct SQL access** - We've introduced support for [SQL](https://xata.io/docs/sdk/sql/overview) (Postgres dialect) over HTTP, with a focus on safety. 
+  - To ensure secure SQL access to Xata databases, we've developed a dedicated SQL proxy. The proxy works alongside the Xata data API, enhancing security between Postgres, routing, and authentication components. 
+  - This proxy thoroughly understands and verifies executed SQL commands, parsing queries, checking access to tables and columns, and monitoring functions.
+  - We have plans to further expand SQL support and introduce the Postgres Wire protocol for maximum compatibility. 
+- **New integrations** - Xata now offers integrations with [LangChain](https://xata.io/docs/integrations/langchain), the ORM [Drizzle](https://xata.io/docs/integrations/drizzle), and the TypeScript builder [Kysley](https://xata.io/docs/integrations/kysley), expanding the range of tools and possibilities for developers.
+- **Xata's new framework guides** - Our getting started guides for the frameworks [Astro](https://xata.io/docs/getting-started/astro), [Next.js](https://xata.io/docs/getting-started/nextjs), [Nuxt](https://xata.io/docs/getting-started/nuxt), [Remix](https://xata.io/docs/getting-started/remix), [SolidStart](https://xata.io/docs/getting-started/solidstart), and [SvelteKit](https://xata.io/docs/getting-started/sveltekit) have undergone a significant overhaul. These getting started guides provide a more user-friendly introduction to Xata, making it easier for users to learn how to use Xata effectively. Use the guides to learn about the Xata CLI, creating databases, defining schemas, importing CSV data, and leveraging querying, filtering, and full-text fuzzy search capabilities.
+
+### Fixes
+
+We've addressed and resolved multiple issues for improved performance and reliability in the last month. 
+
+- 
+
+### Blog posts
```

Comment:
It's been about a month since we had a changelog. We could probably add two more blogs too:

https://xata.io/blog/xatabyte-database-indexing
https://xata.io/blog/database-schema-design